sdk/python/kfp/compiler/_default_transformers.py

```diff
@@ -12,16 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import uuid
+from typing import Dict, Optional, Text
 from ..dsl._container_op import BaseOp, ContainerOp
 
+# Pod label indicating the SDK type from which the pipeline is
+# generated. By default it's set to kfp.
+_SDK_ENV_LABEL = 'pipelines.kubeflow.org/pipeline-sdk-type'
+_SDK_ENV_DEFAULT = 'kfp'
+
+# Pod label representing the random pipeline ID. If two pods have the same
+# pipeline ID then they belong to the same KFP pipeline.
+_PIPELINE_UUID_LABEL = 'pipelines.kubeflow.org/pipeline-uuid'
```

Comment:
This is not the pipeline UUID. When referring to pipeline UUID, there's only one: the one decided/accepted by the apiserver (pipeline UUID + version UUID).
This is adding yet another uuid without it having an actual meaning.

Why not set this on the apiserver and make it the same as the actual pipeline uuid?
Or, why even set this since this information already exists on the workflow object, and the workflow object is referenced on a pod label (Alexey has already mentioned this in another comment).

What's the purpose? Could it be covered by the aforementioned suggestion (apiserver setting)?

Reply:
Resolved by removing the pipeline id here.